robyn/__init__.py

```diff
@@ -39,7 +40,7 @@ def __init__(self, file_object: str) -> None:
         self.headers = []
         self.directories = []
         self.event_handlers = {}
-
+        self.load_var = load_vars()
```

Comment:
One last thing before we can proceed with the tests, `self.load_var` is `None`.  So, we can just call the `load_vars()` function.